docs/source/programming_resources/tutorial_specific/android_studio/fork_and_clone_github_repository/Fork-and-Clone-From-GitHub.rst

```diff
@@ -530,48 +529,55 @@ Tags from remotes are not automatically copied into a repository on a clone.  To
 The --all option fetches at once from all remotes, the --tags option tells git to fetch the tags.
 Tags always follow the semantic versioning rules.  e.g. v7.0, v7.1, v7.2, v8.0, etc.
 
-The ^ syntax allows one to reference parents of a commit and can be applied to tag names.  tag^ is the immediate parent of the commit
+The `^ syntax <https://medium.com/@gabicle/git-ancestry-references-explained-bd3a84a0b821>`_ allows one to reference parents of a commit and can be applied to tag names.  tag^ is the immediate parent of the commit
 tag points to.  For commits with multiple parents such as merge commits one can apply a number to refer to a specific parent.
 tag^1 is the same as tag^ and is the first parent of the commit, tag^2 is the second parent of the commit.
 
 Merging the Inverse of an SDK Update
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-The diagram below shows the v8.0 tag pointing to the v8.0 merge commit along with references to the parents of v8.0.
+The diagram below shows an example repository structure. The `master` branch tracks the upstream `master`, and the 
+`competition` branch contains merges from `master` and commits from the team. Highlighted are the v8.0 tag pointing to the 
+v8.0 merge commit along with references to the parents of v8.0 (v8.0^1 and v8.0^2).
 
    .. figure:: images/tags.png
       :align: center
       :alt: tags
 
       v8.0 tag pointing to the v8.0 merge commit.
 
-.. warning:: If you want to downgrade more than one revision you must revert each revision in sequence otherwise you could wind up
+.. warning:: 
+   If you want to downgrade more than one revision you must revert each revision in sequence otherwise you could wind up
    with changes remaining after reversion from the SDK version in between latest and the target you are referring to.
-   For example if you need to downgrade from v8.1.1 to v8.0 you revert v8.1.1 followed by v8.1.  If you don't follow this order,
-   then changes in v8.1.1 that don't overlap with v8.1 will remain in your workspace and that's not what you want.
+   For example if you need to downgrade from v8.0 to v7.1 you revert v8.0 followed by v7.2.  If you don't follow this order,
+   then changes in v8.0 that don't overlap with v7.2 will remain in your workspace and that's not what you want.
```

Comment:
I changed this example from going "8.1.1 to 8.0" to "8.0 to 7.1" because nowhere in any of these documents do we actually mention or show 8.1.1 and 8.1 - nobody knows if there was an 8.0.1 that might also have to be reverted, or any other references that someone without institutional knowledge would simply "just know."

Reply:
I chose not to use a non-competition legal downgrade example because I didn't think we wanted to imply that this was something that they could do and compete with.  If you really think v8.1.1 is a confusion factor, then I'd suggest a downgrade example from v7.2 to v7.0 for the multi-version downgrade.

Reply:
If you do that, you need to change the diagram to branch from 7.0, include 7.1, and also 7.2. That way they can visualize the path.

Reply:
And then change your example scenario at the end to revert 7.2 singularly back to 7.1, instead of reverting 8.0 back to 7.2.